prog/vnet/load_balancer_nexus.rb

```diff
@@ -142,6 +155,9 @@ def need_to_rewrite_dns_records?
       dns_zone.delete_record(record_name: load_balancer.hostname)
     end
 
+    load_balancer.vm_ports.map(&:destroy)
+    load_balancer.ports.map(&:destroy)
```

Comment:
there is already a plugin association for `ports` in `model/load_balancer.rb`, why do we need this?